README.md

```diff
@@ -279,10 +440,24 @@ app.get("/", function(context) {
 
 <div id="route_use"><h3>use(<i>callback</i>)</h3></div>
```

Comment:
This method parameter can either be a single middleware function or an array of middleware functions